src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -299,6 +299,54 @@ public virtual async Task PatchOne_Update_Test()
                 );
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a computed field is missing from the request body.
+        /// </summary>
```

Comment:
Can you add context to _how_ successful execution validates what you are testing? A `PATCH` request shouldn't touch fields not listed in the request body by definition. Your validation SQL query also does not query the `last_sold_on_date` field to check for value changes. 

An explanation like 'this test would fail when DAB includes the `last_sold_on_date` field in the PATCH request, resulting in an exception being returned by the database due to an attempt to modify a read-only field.'

Reply:
added explanation to tests.